docs/application-development-guide.md

```diff
@@ -4409,6 +4916,11 @@ const {yourActivity} = proxyActivities<typeof activities>({
 });
 ```
 
+</TabItem>
+<TabItem value="python">
+
+Content is not available
```

Comment:
Related note, I don't think we should be providing this activity retry simulator tool inside dev guide. Just link to the tool somewhere else.